packages/nx-plugin/src/internal/execute-process.ts

```diff
@@ -138,40 +159,71 @@ export type ProcessObserver = {
  * @param cfg - see {@link ProcessConfig}
  */
 export function executeProcess(cfg: ProcessConfig): Promise<ProcessResult> {
-  const { observer, cwd, command, args, ignoreExitCode = false } = cfg;
+  const {
+    observer,
+    cwd,
+    command,
+    args,
+    ignoreExitCode = false,
+    env,
+    dryRun,
+  } = cfg;
   const { onStdout, onError, onComplete } = observer ?? {};
   const date = new Date().toISOString();
   const start = performance.now();
 
-  const logCommand = [command, ...(args || [])].join(' ');
   ui().logger.log(
     gray(
-      `Executing command:\n${logCommand}\nIn working directory:\n${cfg.cwd ?? process.cwd()}`,
+      `Executing command:\n${formatCommandLog({
+        command,
+        args: args ?? [],
+        env,
+      })}\nIn working directory:\n${cfg.cwd ?? process.cwd()}`,
     ),
```

Comment:
The `executeProcess` implemented in `utils` now has more compact command formatting that's easier to read.
- #967

https://github.com/code-pushup/cli/blob/c7d314e8c835b0332d3b0ddd1407987621d45179/packages/utils/src/lib/execute-process.ts#L154-L158

https://github.com/code-pushup/cli/blob/c7d314e8c835b0332d3b0ddd1407987621d45179/packages/utils/src/lib/format-command-log.ts#L12-L27

I suggest copying it here, so we don't reinvent the wheel and have consistent command logs across all our tools.